suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -184,6 +184,17 @@ export const selectAccountLabel = memoizeWithArgs(
     },
 );
 
+export const selectAccountNetworkSymbol = memoizeWithArgs(
+    (state: AccountsRootState, accountKey: AccountKey): NetworkSymbol | null => {
+        const account = selectAccountByKey(state, accountKey);
+
+        if (!account) return null;
+
+        return account.symbol;
+    },
+    { size: 25 },
```

Comment:
No need to memoize this as the return will change with each `accountKey` and the memoization is useless.

Reply:
Size arg removed, let's discuss usage of memorization for redux selectors later as agreed.  b9ee3bd35